.buildkite/nightly-benchmarks/serving-tests.json

```diff
@@ -0,0 +1,70 @@
+[
+    {
+        "test_name": "serving_llama8B_tp1_sharegpt",
+        "server_parameters": {
+            "model": "meta-llama/Meta-Llama-3-8B",
+            "tensor_parallel_size": 1,
+            "swap_space": 16,
+            "disable_log_stats": "",
+            "disable_log_requests": "",
+            "load_format": "dummy"
+        },
+        "client_parameters": {
+            "model": "meta-llama/Meta-Llama-3-8B",
+            "dataset_name": "sharegpt",
+            "dataset_path": "./ShareGPT_V3_unfiltered_cleaned_split.json",
+            "num_prompts": 1000
```

Comment:
Do we consider only running this to mimic offline processing (sending 1000 prompts in one shot)? I think since we're doing this nightly benchmark anyways, we should also run at certain QPS as a way to simulate online traffic.

Reply:
Currently the goal for this PR is to run on:
1. After merging a PR
2. All the commits in those PR that contains `perf-benchmarks` label
So this CI will be ran relatively frequently. In this sense, I am planning to only cover the latency (`benchmark_latency.py`) and the throughput (`benchmark_serving.py` with request rate=inf) of vllm.

The following PR (#5362) will focus on building the nightly benchmark for both vllm and other alternatives, and we can add more workloads (like varying QPS) there.